src/CLRunner.ts

```diff
@@ -90,7 +90,7 @@ interface IActionInputLogic {
     logicResult: CLM.LogicResult | undefined
 }
 interface IActionInputRenderOnly {
-    type: ActionInputType
+    type: Exclude<ActionInputType, ActionInputType.RENDER_ONLY>
```

Comment:
I tried to follow the pattern there but these names seem backwards to me. Can you explain?

`IActionInputLogic`'s type is `RENDER_ONLY` 
`IActionInputRenderOnly`'s type is `LOGIC_ONLY` and `LOGIC_AND_RENDER`

Seems like it should be the other way arounds so they match.
`IActionInputRenderOnly` should be `RENDER_ONY` and
`IActionInputLogic` should be `LOGIC_ONLY` and `LOGIC_AND_RENDER`